src/cli/doc_get.rs

```diff
@@ -112,6 +114,25 @@ fn run_get(state: Arc<Mutex<State>>, mut args: CommandArgs) -> Result<OutputStre
         ids.push(id);
     }
 
+    let batch_size = 500;
```

Comment:
can we make that a configurable / static somewhere? so it's not duplicated for each command (i.e. in the user config somewhere)

Reply:
I was going to do that as a later improvement, it should be a user configurable setting - along with max number kv connections when/if we add that.